server/src/main/java/org/spine3/server/command/CommandBus.java

```diff
@@ -103,35 +103,50 @@
     private boolean isMultitenant;
 
     /**
-     * Creates a new instance.
+     * Determines whether the manual thread spawning is allowed within current runtime environment.
      *
-     * @param commandStore a store to save commands
-     * @return a new instance
+     * <p>If set to {@code true}, {@code CommandBus} will be running some of internal processing in parallel
+     * to improve performance.
+     */
+    private final boolean isThreadSpawnAllowed;
```

Comment:
As discussed vocally, let's keep it the way it is because it's consistent with `isMultitenant`. We can revisit two names later, when making `isMultitenant` `final`.